api/routes.go

```diff
@@ -65,4 +65,9 @@ func (s *Server) RegisterRoutes() {
 	r.Post("/tags/delSeries", withOrg, ready, bind(models.GraphiteTagDelSeries{}), s.graphiteTagDelSeries)
 	r.Combo("/functions", withOrg, ready).Get(s.graphiteFunctions).Post(s.graphiteFunctions)
 	r.Combo("/functions/:func(.+)", withOrg, ready).Get(s.graphiteFunctions).Post(s.graphiteFunctions)
+
+	// Prometheus endpointes
+	r.Combo("/prometheus/api/v1/query_range", cBody, withOrg, ready, form(models.PrometheusQueryRange{})).Get(s.prometheusQueryRange).Post(s.prometheusQueryRange)
```

Comment:
for consistency with the PrometheusSeriesQuery, `models.PrometheusQueryRange` should be called `models.PrometheusRangeQuery` (and also because it is a rangequery)